lib/ocpp/v201/connectivity_manager.cpp

```diff
@@ -82,13 +85,31 @@ ConnectivityManager::get_network_connection_profile(const int32_t configuration_
     return std::nullopt;
 }
 
+std::optional<int> ConnectivityManager::get_configuration_slot_priority(const int configuration_slot) {
+    // Convert to string as a vector of strings is used.
+    const std::string configuration_slot_string = std::to_string(configuration_slot);
+
+    auto it = std::find(this->network_connection_priorities.begin(), this->network_connection_priorities.end(),
+                        configuration_slot_string);
+    if (it != network_connection_priorities.end()) {
+        // Index is iterator - begin iterator
+        return it - network_connection_priorities.begin();
+    }
+    return std::nullopt;
+}
+
+const std::vector<std::string>& ConnectivityManager::get_network_connection_priorities() const {
+    return network_connection_priorities;
```

Comment:
It's not an official rule but to keep style (also some other places):
```suggestion
    return this->network_connection_priorities;
```